site/Tables.csv

```diff
@@ -1,21 +1,21 @@
-Instrument,Instrument,Instrument,Instruments that are used for measures in WWMeasure and SiteMeasure. The assay method for viral measurement are described in AssayMethod.,"L'instrument utilisé pour mesurer les échantillons et l'eau usée des sites d'échantillonnage. La méthode d'analyse pour les mesures virales sont décrites dans la table ""AssayMethod""."
-Polygon,Area,Domaine,"A simple polygon that encloses an area on the surface of the earth, normally these polygons will either be of a sewer catchment area or of a health region or other reporting area.","Polygone englobant une région de la surface terrestre. Normalement, ces polygones représentent soit des bassin de drainage d'un réseau d'égouts ou une région de santé publique, ou une autre région associée à des données reportées."
-Lookup,Lookup Table,Tableau de recherche,Used for lookup values of all category based columns,Utiliser pour retrouver les valeurs associées aux colonnes contenant des données catégoriques.
-Assay,Assay,Essai,Procedure that yields measures.,Procédé qui produit des mesures.
-AssayHasInstrument,Assay has instrument,Essai utilise instrument,Shows relations between assays and experiments.,Montre les relation entre les essais et les instruments
-Domain,Domain,Domaine,"Largest measure category (chemical, biological, etc.).","Classification des mesures la plus générale (biologique, chimique, etc.)."
-Measure,Measure,Mesure,Definition of measured quantity or quality.,Définiton d'une quantité ou qualité mesurée.
-MeasureClass,Measure class,Classe de mesure,Group of measures that are performed using similar assay methods.,Classe de mesures utilisant des procédures d'essai similaires.
-MeasureClassHasUnit,Measure class has unit,Classe de mesure utilise unité,Shows relations between a measure class and reporting units.,Montre les relations entre les classes de mesures et les unités utilisées pour raporter leur valeur.
-MeasureGroup,Measure group,Groupe de mesure,Groups measures that have relate to the same.,
-UnitScale,Unit Scale,Échelle des unités,"Cardinality of a measurement (qualitative, quantitative, etc.).","Cardinalité d'une mesure (quantitative, qualitative, etc.)."
-Specimen,Specimen,Spécimen,Class of entity from which a measure is taken.,"Classe d'entité sur laquelle une mesure est prise (échantillon, site, etc.)."
-Unit,Unit of measure,Unité de mesure,Unit of reporting for values,Unité pour rapporter des valeurs
-UnitHasAggregationScale,Unit has aggregation scale,Unité utilise échelle d'aggrégation,Aggregations that are meaningful for a given unit,Aggrégations qui sont utilisables avec une unité
-AggregationScale,Aggregation scales,Échelle d'aggrégation,Scale used to aggregate values together,
\ No newline at end of file
+tableName,tableShortFormalName,tableLabel,tableDesc

```

Comment:
Looks good. I wonder if we should have `Sa` as opposed to `sa` to keep the Pascal case approach for the original table.

Reply:
I would say I'm pretty agnostic on the question of capitalization - my thought process for all lowercase is that capitalization in Pascal Case or Camel Case seems to be to improve readability in the absence of spaces, but I don't know that the UIDs are all that readable regardless. Do you think I should update them all with a capitalized first letter?